### PR TITLE
secure/updatePlace added

### PR DESCRIPTION
**Pull request by Devang47**

### CODE REVIEW

**Review by BlowaterNostr — CHANGES_REQUESTED** [review]
@Devang47 Thanks for submitting the first PR. Well done. This codebase is the bridge between the client and the backend. Therefore, please add an intergration test. Look at `tests` dir for examples.

File: tests/authed_api.test.ts [hunk 1]
```diff
@@ -122,3 +120,24 @@ Deno.test("post notes", async () => {
...
+        const result = await client.updatePlace({
+            id: 23751,
+            name: "Florianópolis",
+        });
+        if (result instanceof Error) {
+            fail(result.message);
+        }
+        assertEquals(result.id, 23751);
+    }
+});
```
[the remark this answers is not in the document]
Comment (alberlantis) [reply tests/authed_api.test.ts:143]: A good way to test an `update` function is to
1. get a place 
2. update this place's name to some random data
3. get it again, verify it is changed to this random data
4. set it back
5. get it again and verify it is back to the original data

**Review by alberlantis — CHANGES_REQUESTED** [review]
@Devang47 Please remember to `rebase` as well

File: tests/authed_api.test.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (Devang47) [reply tests/authed_api.test.ts:143]: Done

**Pull request merged**